### PR TITLE
PODS-10193 - Removed toggle related code for aft

### PR DESCRIPTION
**Pull request by jamdade-sharad**

### CODE REVIEW

File: app/controllers/partials/PspSchemeDashboardPartialsController.scala [hunk 1]
```diff
@@ -59,8 +57,7 @@ class PspSchemeDashboardPartialsController @Inject()(
               val futureSeqHtml = for {
                 schemeDetails <- schemeService.retrieveSchemeDetails(request.idOrException, idNumber, "srn")
                 schemeFs <- financialStatementConnector.getSchemeFS(schemeDetails.pstr)
-                interimDashboard <- featureToggleConnector.getNewPensionsSchemeFeatureToggle("interim-dashboard").map(_.isEnabled)
-                aftReturnsHtml <- pspDashboardAftReturnsPartial(interimDashboard, idNumber, schemeDetails.pstr, psaId)
+                aftReturnsHtml <- Future.successful(Html(""))
```
Comment (PavelVjalicin) [inline app/controllers/partials/PspSchemeDashboardPartialsController.scala:60]: Remove aftReturnsHtml entirely.
Should just yield scala.collection.immutable.Seq(paymentsAndChargesHtml)
I think this should work.

**Review by PavelVjalicin — CHANGES_REQUESTED** [review]
Minor comment

File: app/controllers/partials/PspSchemeDashboardPartialsController.scala (hunk 1 above)
Comment (jamdade-sharad) [reply app/controllers/partials/PspSchemeDashboardPartialsController.scala]: removed

Comment (jamdade-sharad) [reply app/controllers/partials/PspSchemeDashboardPartialsController.scala]: removed

**Review by PavelVjalicin — APPROVED** [review]

**Pull request merged**